packages/openapi-to-graphql/src/types/preprocessing_data.ts

```diff
@@ -42,7 +42,7 @@ export type PreprocessingData = {
   /**
    * List of all the used object names to avoid collision
    */
-  usedOTNames: string[]
+  usedTypeNames: string[]
```

Comment:
This contained not only object type names but also array, enum, and input object type names

Reply:
Makes sense.